Sources/SWBCore/Extensions/SpecificationsExtension.swift

```diff
@@ -69,4 +69,22 @@ extension SpecificationsExtension {
     public func specificationClassesClassic() -> [any SpecClassType.Type] { [] }
     public func specificationImplementations() -> [any SpecImplementationType.Type] { [] }
     public func specificationSearchPaths() -> [URL] { [] }
+
+    public func findResourceBundle(nameWhenInstalledInToolchain: String, defaultBundle: Bundle?) -> Bundle? {
+        #if SEARCH_TOOLCHAIN_FOR_RESOURCES
+        let resourceBundleExtension: String
+        #if canImport(Darwin)
+        resourceBundleExtension = "bundle"
+        #else
+        resourceBundleExtension = "resources"
+        #endif
+        if let executablePath = try? Path(CommandLine.arguments[0]).resolveSymlink(fs: localFS) {
```

Comment:
CommandLine.arguments in libraries is not great, can we instead plumb this down or base it on `core.developerDir`?

Reply:
I don't know of any good alternatives right now, core.developerDir really isn't consistent enough across platforms

Reply:
Actually, this should be toolchain relative. That should already be consistent across platforms, including on Windows and in Xcode. Can you pass in the toolchain registry to specificationFiles() and look it up relative to the default toolchain?

Would require swapping the order of initializeSpecRegistry() and initializeToolchainRegistry(); hopefully that's OK...

Reply:
That doesn't work on linux where the fallback system toolchain exists

Reply:
Why not? On Linux the path of the fallback system toolchain is going to be the path 3 directories up from where the `swift` executable lives, so I think that isn't any different from other platforms. You should be able to construct `toolchain.path.join("share")` to get to the resources location...

Reply:
I ended up exposing API for search paths here instead, so SwiftPM can point us to its own resource dir